src/test/e2e-emulated/contracts/maps.tact

```diff
@@ -945,4 +945,9 @@ contract MapTestContract {
     get fun intMap1Del(key: Int): Bool {
         return self.intMap1.del(key);
     }
+
+    get fun checkNullReference(): Int {
+        let m: map<Int, Int> = emptyMap();
+        return m.get(0)!!;
```

Comment:
I just checked with the online IDE and it actually throws exit code 128 in this getter. The IDE runs Tact v1.4.0. This needs to be a receiver, because getters are not supposed to change the contract's state, so it looks like the semantics of `impure` is a bit different in the FunC compiler for these two cases.